packages/custom-functions-metadata-plugin/package.json

```diff
@@ -26,7 +26,8 @@
   "dependencies": {
     "custom-functions-metadata": "^1.0.48",
     "es6-promise": "^4.2.8",
-    "webpack": "^5.24.4"
+    "webpack": "^5.24.4",
+    "loader-utils": "*",
```

Comment:
This ```,``` needs to be removed. Hopefully you can build the project by now to detect those kind of errors